### PR TITLE
Testing/ak/search

### LINKED ISSUE #33: Add testing to search component
- [X] Make sure search bar is on page
- [X] Check search bar is clear after button click

### PR DESCRIPTION
**Pull request by amykarnaze**
#### What's this PR do?
- Adds 1 test to see if search bar is rendered on page and another to check inputs are clear upon button click.

Type of change made:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [X] Testing

#### Where should the reviewer start?
- Search.test.js

#### Why is this change required? What problem does it solve?
- Checks to make sure the correct content is rendered on the page.

#### What are the relevant tickets?
- Closes #33

### CODE REVIEW

File: src/Components/Search/Search.test.js [hunk 1]
```diff
@@ -1,4 +1,34 @@
 import React from 'react'
 import '@testing-library/jest-dom'
-import { screen, render, waitFor } from '@testing-library/react'
-import Search from './Search.js'
\ No newline at end of file
+import { screen, render, waitFor, fireEvent } from '@testing-library/react'
+import Search from './Search.js'
+jest.mock('../../API');
```
Comment (NicoleLatifi) [inline src/Components/Search/Search.test.js:5]: Will we need to mock the API requests to test the search component?

**Review by NicoleLatifi — APPROVED** [review]

File: src/Components/Search/Search.test.js (hunk 1 above)
Comment (NakiNorton) [reply src/Components/Search/Search.test.js:5]: We could also create a mock store to hold a book to search for. Then test that the book is being displayed?

**Review by NakiNorton — APPROVED** [review]

File: src/Components/Search/Search.test.js [hunk 2]
```diff
@@ -1,4 +1,34 @@
...
+
+describe('Search Componey', () => {
```
Comment (kip-west) [inline src/Components/Search/Search.test.js:7]: nitpick: small spelling thing! should be `Component`

File: src/Components/Search/Search.test.js [hunk 3]
```diff
@@ -1,4 +1,34 @@
...
+    const submitButton = screen.getByRole('button', {name: 'Search'})
+
+    expect(searchInput).toBeInTheDocument()
+    expect(submitButton).toBeInTheDocument()
+
+    fireEvent.change(searchInput, {target: {name: 'search', value: 'Anything'}})
+    expect(searchInput.value).toEqual('Anything')
+    fireEvent.click(submitButton)
+    
+    expect(searchInput.value).toHaveLength(0)
```
Comment (kip-west) [inline src/Components/Search/Search.test.js:32]: Great job getting these tests going for the search functionality!! This looks great.

**Review by kip-west — APPROVED** [review]

**Pull request merged**